machine-agent/templates/service.yaml

```diff
@@ -10,9 +10,9 @@ metadata:
 spec:
   ports:
     - port: {{ .Values.analytics.port }}
-      targetPort: http
+      targetPort: {{ .Values.analytics.port }}
       protocol: TCP
-      name: http
+      name: analytics
   selector:
-    app.kubernetes.io/name: {{ include "machine-agent.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
+    app: {{ include "machine-agent.name" . }}
+    release: {{ .Release.Name }}
```

Comment:
Same thing here,
We should change the labels in daemonSet.yaml as it makes sense to have group based labels

Reply:
Labels were reverted to what they were in the service selector and updated on the daemonset instead of vice versa.